packages/suite/src/utils/suite/device.ts

```diff
@@ -116,7 +116,7 @@ export const getVersion = (device: TrezorDevice) => {
     return features && features.major_version > 1 ? 'T' : 'One';
 };
 
-export const getFwVersion = (device: AcquiredDevice) => {
+export const getFwVersion = (device: KnownDevice) => {
```

Comment:
why this change?
`AcquiredDevice` is extended `KnownDevice`

Reply:
`KnownDevice` always have `features, so it would be better to have here a more specific type.

I wanted to use it with `prevDevice` which is in fact `Device` from `DEVICE.DISCONNECT` event. I check, that it has `features` so the possibility to be `UnknownDevice` is not valid anymore.

https://github.com/trezor/trezor-suite/pull/3818/files/6984f5fe00bab95510e3a04c1225628a87a16c2a#diff-545d00698b46c7b1a497151825b00ef48aea064f955cf81b2526e8451dfc9458R57

Reply:
`AcquiredDevice` > `KnownDevice`, but `KnownDevice` is enough for this usecase